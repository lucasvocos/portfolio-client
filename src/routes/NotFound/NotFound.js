import React from 'react'
import { Link } from 'react-router-dom'
import './NotFound.scss'

class NotFoundRoute extends React.Component {
  render() {
    return (
      <section>
        <h2>404 — Sorry, this page isn't found</h2>
        <Link
          className='serif'
        to='/'>
        Back to homepage
        </Link>
      </section>
    );
  }
}

export default NotFoundRoute
