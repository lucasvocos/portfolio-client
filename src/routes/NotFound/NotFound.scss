section {
  padding: 0 20px;
  .serif {
    letter-spacing: 0.0125em;
  }
}
