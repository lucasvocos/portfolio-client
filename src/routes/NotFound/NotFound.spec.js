import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import NotFound from './NotFound'

describe(`NotFound component`, () => {
  it('renders a NotFound by default', () => {
    const wrapper = shallow(<NotFound />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
