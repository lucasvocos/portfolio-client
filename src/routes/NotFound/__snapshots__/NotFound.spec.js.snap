// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component renders a NotFound by default 1`] = `
<section>
  <h2>
    404 — Sorry, this page isn't found
  </h2>
  <Link
    className="serif"
    to="/"
  >
    Back to homepage
  </Link>
</section>
`;
